src/DataFixtures/ContentFixtures.php

```diff
@@ -63,9 +61,11 @@ private function loadContent(ObjectManager $manager): void
             $amount = $contentType['singleton'] ? 1 : (int) ($contentType['listing_records'] * 3);
 
             foreach (range(1, $amount) as $i) {
-                $ref = $i === 1 ? 'admin' : ['admin', 'henkie', 'jane_admin', 'tom_admin'][random_int(0, 3)];
-                /** @var User $author */
-                $author = $this->getReference($ref);
+                if ($i === 1) {
+                    $author = $this->getReference('user_admin');
+                } else {
+                    $author = $this->getRandomReference('user');
```

Comment:
this is not a class name

Reply:
What do you mean?

Reply:
getRandomReference expects a class name as parameter

Reply:
Renamed